### PR TITLE
Remove Null from display when amendment version is unknown

### LINKED ISSUE #2218: Drop null from filings tables
**What we're after:** For filings datatables, we include both `Version unknown` and `null` if we don't know the version or the fec_file_id. In these cases where both are unknown we should only provide one result. "Version unknown" seems sufficient to cover both missing pieces of information, but null is a term we try to avoid. 

screen shot 2018-07-26 at 9 39 24 am

**Completion criteria:**
- [x] Confirm "Version unknown" is good with content team
- [x] Identify field that populates "current version, and add logic that displays "Version unknown" in cases when neither that field nor fec_file_id are available
- Make the same change on the following tables: 
  - [x] https://www.fec.gov/data/filings/
  - [x] https://www.fec.gov/data/reports/presidential/
  - [x] https://www.fec.gov/data/reports/house-senate/
  - [x] https://www.fec.gov/data/reports/pac-party/

### PR DESCRIPTION
**Pull request by pkfec**
## Summary (required)

Update the text under version column to display `Version unknown` instead of `Version unknown null` when `most_recent` flag is null.

- Resolves #2218 

### Required reviewers

1 content and 2 or more developers

## Impacted areas of the application

-  Candidate filings section, Committee filings section, Filings, Presidential reports, house-senate and pac-party report data tables.

## Screen shots:
Example: PAC and party committee reports (https://www.fec.gov/data/reports/pac-party/?two_year_transaction_period=1988&data_type=processed&min_receipt_date=01%2F01%2F1987&max_receipt_date=12%2F31%2F1988)

**Before:**
Screen Shot 2023-05-24 at 9 56 51 AM

**Now:**
Screen Shot 2023-05-24 at 9 58 45 AM

**From Transaction Time period > 2000** , there are cases when Version is Unknown and yet have `fec_file_id` 

Screen Shot 2023-05-30 at 11 47 21 AM

## How to test

- `git checkout` `feature/2218-drop-null-filing-table`
- `cd fec`
- `npm run build-js`
- run `pytest`
- run `./manage.py runserver`
- Test following URLs in browser

| Website sections | Prod URL's | Local URL's|
| ------   | ------  | ------  | 
|Candidate filings | https://www.fec.gov/data/candidate/P80003338/?tab=filings | http://localhost:8000/data/candidate/P80003338/?tab=filings|
|Committee filings | https://www.fec.gov/data/committee/C00000935/?tab=filings&cycle=1988| http://localhost:8000/data/committee/C00000935/?tab=filings&cycle=1988 |
|Filings | https://www.fec.gov/data/filings/?data_type=processed (sort by receipt_date ascending)| http://localhost:8000/data/filings/?data_type=processed (sort by receipt_date ascending)|
|Presidential committee reports | https://www.fec.gov/data/reports/presidential/?data_type=processed (sort by receipt_date ascending)  | http://127.0.0.1:8000/data/reports/presidential/?data_type=processed (sort by receipt_date ascending )|
|House and Senate committee reports | https://www.fec.gov/data/reports/house-senate/?data_type=processed (sort by receipt_date ascending)| http://localhost:8000/data/reports/house-senate/?data_type=processed (sort by receipt_date ascending) |
|PAC and party committee reports | https://www.fec.gov/data/reports/pac-party/?two_year_transaction_period=1988&data_type=processed&min_receipt_date=01%2F01%2F1987&max_receipt_date=12%2F31%2F1988 (sort by receipt_date ascending)| http://localhost:8000/data/reports/pac-party/?two_year_transaction_period=1988&data_type=processed&min_receipt_date=01%2F01%2F1987&max_receipt_date=12%2F31%2F1988 (sort by receipt_date ascending) |

### CODE REVIEW

**Review by johnnyporkchops — APPROVED** [review]
Tested locally on datatables, works great. Thanks @pkfec

File: fec/fec/static/js/modules/columns.js [hunk 1]
```diff
@@ -37,9 +37,7 @@ var versionColumn = {
     var version = helpers.amendmentVersion(data);
     if (version === 'Version unknown') {
       return (
-        '<i class="icon-blank"></i>Version unknown<br>' +
-        '<i class="icon-blank"></i>' +
-        row.fec_file_id
+        '<i class="icon-blank"></i>Version unknown<br>'
       );
     } else {
       if (row.fec_file_id !== null) {
```
Comment (JonellaCulmer) [inline fec/fec/static/js/modules/columns.js:43]: Will this still show null?

Comment (JonellaCulmer) [reply fec/fec/static/js/modules/columns.js:43]: Thanks, @pkfec!

Comment (dorothyyeager) [reply fec/fec/static/js/modules/columns.js:43]: Removing myself as reviewer, this really only needs one content reviewer.

**Review by kathycarothers — APPROVED** [review]

**Comment by fec-jli** [comment]
logic can simplify like this:
```    var version = helpers.amendmentVersion(data);
    if (row.fec_file_id !== null) {
        version = version + '<br><i class="icon-blank"></i>' + row.fec_file_id;
    }
    return version;
```

**Pull request merged**